src/noflo-component-factory.js

```diff
@@ -38,6 +38,7 @@ module.exports = function(nodeDef, callback){
             inPorts: _.mapObject(nodeDef.inPorts, changeMulti2Addressable)
         });
         node.isTranslator = _.isUndefined(nodeDef.isTranslator) ? false : nodeDef.isTranslator;
+        node.isTransient = _.isUndefined(nodeDef.isTransient) ? false : nodeDef.isTransient;
```

Comment:
I think I made the change you are requesting, which does make sense since both isTranslator and isTransient are pipeline flags, not core noflo.    Please take a look and let me know.